org.mozilla.FirefoxNightly/org.mozilla.FirefoxNightly.json

```diff
@@ -5,6 +5,16 @@
     "sdk": "org.gnome.Sdk",
     "command": "firefox",
     /*"tags": [""],*/
+    "add-extensions": {
+        "org.mozilla.FirefoxNightly.flashplayer": {
+	    "directory": "lib/mozilla/plugins",
+	    "add-ld-path": "lib",
```

Comment:
Can plugins really include additional shared objects into the library path? This doesn't seem needed.